data-types.md

```diff
@@ -3,4 +3,408 @@ title: Data Types
++------+---------------------------------+
+|  1   | 2016-03-26 15:10:10 +0000 +0000 |
+|  2   | 2016-03-26 00:00:00 +0000 +0000 |
++------+---------------------------------+
+# Note that the first timestamp is UTC-05:00, which is the equivalent of EST.
+~~~
+
+## `INTERVAL`
+
+The `INTERVAL` type stores a value that represents a span of time. 
+
+#### Format
+
+When declaring an `INTERVAL`, format it as `INTERVAL '2h30m30s'`, where the following units can be specified either as positive or negative decimal numbers:
+
+- `h` (hour)
+- `m` (minute)
+- `s` (second)
+- `ms` (millisecond)
+- `us` (microsecond)
+- `ns` (nanosecond)
+
+Alternately, you can format an interval as follows:
+
+- `'12h2m1s23ms'::INTERVAL`
+- `CAST ('12h2m1s23ms' AS INTERVAL)`
+
+Regardless of the units used, the interval is stored as hour, minute, and second, for example, `12h2m1.023s`.
+
+#### Examples
+
+~~~
+CREATE TABLE intervals (a INT PRIMARY KEY, b INTERVAL);
+
+SHOW COLUMNS FROM intervals;
++-------+----------+-------+---------+
+| Field |   Type   | Null  | Default |
++-------+----------+-------+---------+
+| a     | INT      | false | NULL    |
+| b     | INTERVAL | true  | NULL    |
++-------+----------+-------+---------+
+
+INSERT INTO intervals VALUES (1111, INTERVAL '2h30m50ns'), (2222, '-2h30m50ns':INTERVAL);
+
+SELECT * FROM intervals;
++------+-------------------+
+|  a   |         b         |
++------+-------------------+
+| 1111 | 2h30m0.00000005s  |
+| 2222 | -2h30m0.00000005s |
++------+-------------------+
+~~~
+
+## `STRING`
+
+The `STRING` type stores strings of variable, unlimited length. 
+
+#### Synonyms 
+
+In CockroachDB, the following are synonyms of `STRING` and are implemented identically: 
+
+- `CHAR` 
+- `CHAR(n)` 
+- `VARCHAR`
+- `VARCHAR(n)` 
+- `TEXT`
+
+#### Length
+
+Length is always variable and unlimited for a `STRING` value. If you specify a fixed length by declaring `CHAR(n)` or `VARCHAR(n)` on a column, it will not be enforced.
+
+#### Format
+
+When declaring a `STRING`, format it as `'a1b2c3'`.
+
+#### Examples
+
+~~~
+CREATE TABLE strings (a STRING PRIMARY KEY, b CHAR, c TEXT);
+
+SHOW COLUMNS FROM strings;
++-------+--------+-------+---------+
+| Field |  Type  | Null  | Default |
++-------+--------+-------+---------+
+| a     | STRING | false | NULL    |
+| b     | STRING | true  | NULL    |
+| c     | STRING | true  | NULL    |
++-------+--------+-------+---------+
+
+INSERT INTO strings VALUES ('a1b2c3', 'd4e5f6', 'g7h8i9');
+
+SELECT * FROM strings;
++--------+--------+--------+
+|   a    |   b    |   c    |
++--------+--------+--------+
+| a1b2c3 | d4e5f6 | g7h8i9 |
++--------+--------+--------+
+~~~
+
+## `BYTES`
+
+The `BYTES` type stores binary strings of variable, unlimited length.
+
+#### Synonyms
+
+In CockroachDB, the following are synonyms of `BYTES` and are implemented identically: 
+
+- `BYTEA` 
+- `BLOB` 
+
+#### Format
+
+When declaring a `BYTES` string, format it as `'a1b2c3'`.
```

Comment:
That's still a `STRING`, not a `BYTES`. Sometimes you can use them interchangeably, but not always. A better way to make `BYTES` is to put a `b` in front of the quotes: `b'a1b2c3'`